netclient/gui/components/views/networks.go

```diff
@@ -49,6 +50,24 @@ func GetNetworksView(networks []string) fyne.CanvasObject {
 				leave(*network)
 			}, components.Danger_color),
 		)
+		cfg, err := config.ReadConfig(*network)
+		if err != nil {
+			fmt.Println(err)
+		}
+		if cfg.Node.Connected == "yes" {
+			grid.Add(
+				components.ColoredIconButton("connection", theme.CheckButtonCheckedIcon(), func() {
```

Comment:
ColoredIconButton("disconnect"

would be better to call it "disconnect" so if you click it it will disconnect.

Reply:
Ok